.github/workflows/aws_openshift_rosa_hcp_single_region_tests.yml

```diff
@@ -0,0 +1,646 @@
+---
+name: Tests - Integration - AWS OpenShift ROSA HCP Single Region
+
+# description: This workflow perform integration tests against ROSA HCP platform
+
+on:
+    schedule:
+        - cron: 0 3 * * 1,3,5 # Runs at 3 AM on Monday, Wednesday, and Friday
```

Comment:
I know it's 8.6 and we have a separate scheduler for this.
But I guess we would not run it 3 times a week, or?
Afaik we run most stuff on a weekly basis.